screen-wake-lock/wakelock-onrelease.https.html

```diff
@@ -20,4 +20,20 @@
     assert_false(ev.cancelable);
   });
 }, "Test onreleased event's basic properties");
+
+promise_test(async t => {
+  await test_driver.set_permission({ name: 'screen-wake-lock' }, 'granted', false);
+
+  const lock = await navigator.wakeLock.request("screen");
+
+  let releaseFired = false;
+  lock.onrelease = t.step_func(() => {
```

Comment:
Nit: do we *really* need that `t.step_func(`? Seems to just add indirection, but ok if you think we need it.

Reply:
We do, otherwise assertion failures will translate into a test harness failure (it's also documented in https://web-platform-tests.org/writing-tests/testharness-api.html#promise-tests).